CHANGELOG.md

```diff
@@ -4,55 +4,76 @@ Notable changes since version 42.0.0, read the complete [History of Changes](htt
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+
 ### Changed
 ### Added
 ### Fixed
 
+## [42.6.1] (2023-11-14 08:05:00 -0500)
```

Comment:
I guess `feat: support SET statements combining with other queries with semicolon in PreparedStatement #2973` means the upcoming version should be `42.7.0`

Reply:
fair enough, will change